pyQuARC/code/schema_validator.py

```diff
@@ -169,6 +176,6 @@ def run(self, xml_metadata, json_metadata):
             (dict): Result of the validation from xml and json schema validators
         """
         return {
-            ** self.run_json_validator(json_metadata)["errors"],
+            **(self.run_json_validator(json_metadata)["errors"] if self.metadata_format == ECHO10 else {}),
```

Comment:
wy?

Reply:
because we don't have a json schema file for dif10. json schema file is the schema file we wrote ourselves. i don't think we need it anymore since we're doing schema validation using `.xsd` files already and it has enough information. should i remove the whole json schema validator thingy?

Reply:
Yes, if there is no dependency there.

Reply:
e09c2cd